src/views/FlawSearchView.vue

```diff
@@ -58,9 +58,7 @@ function saveFilter() {
     },
     {} as Record<string, string>,
   );
-  if (query?.value) {
-    searchStore.saveFilter(filters, query.value);
-  }
+  searchStore.saveFilter(filters, query.value);
```

Comment:
Is this the fix for OSIDB-3450

Reply:
Oh didn't know you created a ticket. That is fix in this PR but in other change:

`...(searchStore.queryFilter ? { query: searchStore.queryFilter } : {}),`

The removal of the condition you are referring is just not necessary so I removed it.